config/database.yml

```diff
@@ -17,14 +17,15 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  password: alfred
+  username: postgres
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
 
 development:
   <<: *default
   database: Anity_Jobs_development
-
```

Comment:
The changes in this file are specific to your local environment and should be added while creating the commit

Reply:
@Tresor11  can you have a deeper look at this?
![Screenshot from 2022-10-21 18-52-47](https://user-images.githubusercontent.com/77842237/197237962-3a919861-620d-46bb-be34-98bf6d9b16de.png)